### ISSUE TITLE
Support Keepalived 1.3+ syntax (and/or version policy?)

### ISSUE THREAD

**Issue by astj** [yuuki/gokc#15]
Recent keepalived changes/adds keepalived.conf syntax, but gokc does not support them.
Do you have any support policy?  (Maybe simply add new version's syntax may cause problem when a user intends to check old version's configuration...)

**Comment by yuuki** [comment]
Yes, gokc will receive patches for newer keepalived syntax in the future.
However, your concern (the conflict between old version's configuration and new version's one) is right.
Ideally, gokc should provide an option to select the version of keepalived you are using such as `gokc --keepalived-version 1.3.0`.
